app/controllers/application_controller.rb

```diff
@@ -0,0 +1,18 @@
+require 'pry'
+require './config/environment'
+
+class ApplicationController < Sinatra::Base
+
+  get "/" do
+    conn = Faraday.new(url: "http://api.openweathermap.org")
+
+    response = conn.get("/data/2.5/onecall") do |req|
+      req.params["lat"] = "35"
+      req.params["lon"] = "139"
+      req.params["appid"] = ENV['WEATHER_API_KEY']
+    end
```

Comment:
This looks great! Eventually we'll be able to pass in params to this endpoint and dynamically set **"lat"** and **"lon"**.

Reply:
yea,